src/main/java/com/stripe/model/Invoice.java

```diff
@@ -450,4 +459,8 @@ public Invoice voidInvoice(Map<String, Object> params, RequestOptions options)
     List<String> lineItemIds;
     Long usageGte;
   }
+
+  public static class TransferData extends StripeObject {
+    String destination;
```

Comment:
whoops i think we might need these magic to make the fields accessible.. 
for setter just for consistency. Same for the `Subscription#TransferData` too.

```
  @Getter
  @Setter
  @EqualsAndHashCode(callSuper = false)
```